tests/jenkins/tests/test_view_single_result.py

```diff
@@ -9,7 +9,7 @@
 def test_open_single_result(base_url, selenium):
     page = TreeherderPage(selenium, base_url).open()
     assert len(page.result_sets) > 1
-    result_set = random.choice(page.result_sets)
+    result_set = random.choice([r for r in page.result_sets if any(r.jobs)])
```

Comment:
I think this can be `if r.jobs`? From what I can tell, `jobs` will either be an empty list, or else contain jobs that are non-falsey - and so `any()` isn't required?

Reply:
You are correct - let me fix that.

Reply:
Fixed, thanks!